server/src/main/java/org/opensearch/wlm/QueryGroupHelper.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.wlm;
+
+import org.opensearch.search.ResourceType;
+import org.opensearch.tasks.Task;
+
+import java.util.Map;
+import java.util.function.Function;
+
+/**
+ * Helper class for calculating resource usage for different resource types.
```

Comment:
Lets correct this comment to convey that this will only contain static helper methods related to QueryGroup feature